t/44-scripts.t

```diff
@@ -16,7 +16,8 @@
 
 use Test::Most;
 
-use FindBin '$Bin';
+use FindBin;
+use lib "$FindBin::Bin/lib";
```

Comment:
Interestingly this causes a compliation error even though we do exactly the same in `03-auth.t` 🤔

Reply:
that part is already covered by a different PR, rebased and resolved.